src/main/java/tutorspet/ui/LessonCard.java

```diff
@@ -0,0 +1,79 @@
+package tutorspet.ui;
+
+import static tutorspet.model.lesson.Lesson.TIME_FORMATTER;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import tutorspet.model.lesson.Lesson;
+
+/**
+ * A UI component that displays information of a {@code Lesson}.
+ */
+public class LessonCard extends UiPart<Region> {
+
+    private static final String FXML = "LessonCard.fxml";
+
+    public final Lesson lesson;
+
+    @FXML
+    private HBox lessonCard;
+    @FXML
+    private Label index;
+    @FXML
+    private Label dayAndTime;
+    @FXML
+    private Label venue;
+    @FXML
+    private Label occurrence;
+
+    /**
+     * Creates a {@code Lesson} with the given {@code lesson} and index to display.
```

Comment:
I think I spotted a typo in the other `Card`s as well, could you help to change them as well?
```suggestion
     * Creates a {@code LessonCard} with the given {@code lesson} and index to display.
```